styles.css

```diff
@@ -0,0 +1,81 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2vh 5vw;
+}
+
+.vector_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.welcome {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.vector {
+  fill: rgb(0, 0, 0);
+  stroke-width: 1;
+  stroke: rgb(0, 0, 0);
+}
+
+.vector:hover {
+  fill: rgb(55, 12, 224);
+  stroke-width: 1;
+  stroke: rgb(55, 12, 224);
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: space-between;
+  height: 100vh;
+}
+
+.pageSection {
+  width: 50vw;
+  flex: 1;
+}
```

Comment:
- [ ] You should give you icons, the same size similar to the one provided from the design.